tensorflow_encrypted/tensor/native.py

```diff
@@ -0,0 +1,296 @@
+from __future__ import absolute_import
```

Comment:
I think this entire guy was renamed to `PrimeTensor`, @justin1121 ?

Reply:
Yep you got it. Probably just want to remove this file.